### ISSUE TITLE
[Bug] Null reference exception when Web API initialized with delegates but without configuration and called from an event handler

### ISSUE THREAD

**Issue by jmprieur** [AzureAD/microsoft-identity-web#1615]
**Which version of Microsoft Identity Web are you using?**
1.22.2

**Where is the issue?**

* Web API
    * [x] Protected web APIs call downstream web APIs

**Is this a new or an existing app?**
New scenario, not met until now (my apps)
Web API initialized with delegates but without configuration and called from an event handler.

**Repro**

Startup.cs

```CSharp
services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
          .AddMicrosoftIdentityWebApi(options => { Configuration.Bind.("AzureAD", options);}, options => {Configuration.Bind.("AzureAD", options);})
            .EnableTokenAcquisitionToCallDownstreamApi(options => {Configuration.Bind.("AzureAD", options);})
               .AddDownstreamWebApi()
            .AddInMemoryTokenCaches();
```

and then `ITokenAcquisition` called from an event handler.

**Expected behavior**
Should work

**Actual behavior**
A null reference exception because the configuration is not found

**Comment by jmprieur** [comment]
Will release in 1.22.3

**Comment by jennyf19** [comment]
fixed in [1.22.3 release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/1.22.3)
